### ISSUE TITLE
How to calcuate execution time on CPU and GPU for for quantized model?

### ISSUE THREAD

**Issue by Afef00** [gewuek/flower_classification_dnndk_v1#1]
Hello,

After running evaluate_quantized_graph.py I wanted to measure the execution time on CPU and GPU but I couldn't find how to do it ?  Any help please? Also I wanted to know how to change the execution of this script fro CPU to GPU?

Thanks!

**Comment by gewuek** [comment]
Hello Afef00,

As a rough test you may just add timestamps before and after the session.run function(loss = sess_loss.run(loss)).
Or you can try some advanced debug flow like https://www.tensorflow.org/guide/profiler but I am afraid that I haven't done that before.
And for Q2, you may need to install the GPU version of TensorFlow.
You may find some useful information here:
https://www.tensorflow.org/install/pip

**Comment by gewuek** [comment]
Hello Afef00,

Since there is no further question, I am going to close this issue.

Thanks
